erpnext/stock/serial_batch_bundle.py

```
@@ -1432,7 +1432,18 @@ def get_series(partial_series, digits):
+			try:
+				frappe.db.bulk_insert("Serial No", fields=fields, values=set(serial_nos_details))
+			except Exception as e:
+				if e and len(e.args) > 0 and "Duplicate" in e.args[1]:
+					frappe.throw(
+						_(
+							"A naming series conflict occurred while creating serial numbers. Please change the naming series for the item {0}."
+						).format(bold(self.item_code)),
+						title=_("Duplicate Serial Number Error"),
+					)
+				else:
+					raise e
```

Ruff (0.14.7)

Use `raise` without specifying exception name

Remove exception name

(TRY201)